uisdk/src/main/java/com/karhoo/uisdk/notification/rides/past/RideNotificationView.kt

```diff
@@ -62,9 +64,14 @@ class RideNotificationView : RideNotificationContract.View {
         return contentView
     }
 
+    @SuppressLint("UnspecifiedImmutableFlag")
```

Comment:
I this needed since you've added the PendingIntent.FLAG_IMMUTABLE ?

Reply:
i left there an if for the < Android 12 devices and i put the suppress for the else case

Reply:
We probably need to update the compileSDKVersion to 31 as well?